safe/impact_statistics/aggregator.py

```diff
@@ -139,7 +139,9 @@ def __init__(self, extent, aggregation_layer):
         # aggregation polygons (one list for one polygon)
         self.impact_layer_attributes = []
 
-        self.processing = processing
+        # Notes(Ismail): Need to initialize Processing in QGIS 2.8.x
```

Comment:
For supporting older versions it may be good to put this in a QGIS version switch (I dont know if initialised call is available in < 2.8)